src/plugins/intel_cpu/src/nodes/kernels/jit_base.hpp

```diff
@@ -0,0 +1,199 @@
+        for (const auto& xmm : xmms) {
+            if (xmm.isXMM()) {
+                uni_vmovdqu(ptr[rsp + offset], xmm);
+                offset += xmm_len;
+            } else if (xmm.isYMM()) {
+                uni_vmovdqu(ptr[rsp + offset], Xbyak::Ymm{xmm.getIdx()});
+                offset += ymm_len;
+            } else if (xmm.isZMM()) {
+                uni_vmovdqu(ptr[rsp + offset], Xbyak::Zmm{xmm.getIdx()});
+                offset += zmm_len;
+            }
+        }
+        add(rsp, get_regs_size(xmms));
+    }
+
+    void uni_vaddps(const Xbyak::Xmm &x,
+                    const Xbyak::Xmm &op1,
+                    const Xbyak::Operand &op2) {
+        if (is_valid_isa(x64::avx))
+            vaddps(x, op1, op2);
+        else {
+            if (x.getIdx() == op1.getIdx()) {
+                addps(x, op2);
+            } else if (x.isEqualIfNotInherited(op2)) {
+                addps(x, op1);
+            } else {
+                movups(x, op1);
+                addps(x, op2);
+            }
+        }
+    }
+
+    void uni_vaddps(const Xbyak::Ymm &x,
+                    const Xbyak::Ymm &op1,
+                    const Xbyak::Operand &op2) {
+        vaddps(x, op1, op2);
+    }
+
+    void uni_vsubps(const Xbyak::Xmm &x,
+                    const Xbyak::Xmm &op1,
+                    const Xbyak::Operand &op2) {
+        if (is_valid_isa(x64::avx)) {
+            vsubps(x, op1, op2);
+        } else {
+            if (x.getIdx() == op1.getIdx()) {
+                subps(x, op2);
+            } else if (x.isEqualIfNotInherited(op2)) {
+                push(op1);
+                subps(op1, op2);
+                movups(x, op1);
+                pop(op1);
+            } else {
+                movups(x, op1);
+                subps(x, op2);
+            }
+        }
+    }
+
+    void uni_vsubps(const Xbyak::Ymm &x,
+                    const Xbyak::Ymm &op1,
+                    const Xbyak::Operand &op2) {
+        vsubps(x, op1, op2);
+    }
+
+    void uni_vcmpps(const Xbyak::Xmm &x,
+                    const Xbyak::Xmm &op1,
+                    const Xbyak::Operand &op2,
+                    const int cmp_predicate) {
+        if (is_valid_isa(x64::avx)) {
+            vcmpps(x, op1, op2, cmp_predicate);
+        } else {
+            if (x.getIdx() == op1.getIdx()) {
+                cmpps(x, op2, cmp_predicate);
+            } else if (x.isEqualIfNotInherited(op2)) {
+                push(op1);
+                cmpps(op1, op2, cmp_predicate);
+                movups(x, op1);
+                pop(op1);
+            } else {
+                movups(x, op1);
+                cmpps(x, op2, cmp_predicate);
+            }
+        }
+    }
+
+    void uni_vcmpps(const Xbyak::Ymm &x1,
+                    const Xbyak::Ymm &x2,
+                    const Xbyak::Operand &op,
+                    const int cmp_predicate) {
+        vcmpps(x1, x2, op, cmp_predicate);
+    }
+
+protected:
+    x64::cpu_isa_t max_cpu_isa_;
```

Comment:
Please don't use such style "name_".

Reply:
Please, let me know which style has to be used here. Should I add m_ prefix or just use camel case? I believe you would expect camel case, but in that case there will be no indication of class member which is too unusual to me, so I would like to have your clarification.